cmd/hivechain/genesis.go

```diff
@@ -163,6 +164,10 @@ func (cfg *generatorConfig) createGenesis() *core.Genesis {
 	// dcc := hexutil.MustDecode("0x" + depositCode)
 	// g.Alloc[dca] = core.GenesisAccount{Code: dcc}
 
+	// Deploy beacon roots storage contract (EIP-4788).
+	asm4788 := common.FromHex("0x3373fffffffffffffffffffffffffffffffffffffffe14604d57602036146024575f5ffd5b5f35801560495762001fff810690815414603c575f5ffd5b62001fff01545f5260205ff35b5f5ffd5b62001fff42064281555f359062001fff015500")
```

Comment:
I think you pasted this incorrectly, there is an extra zero byte at the end here.